index.ts

```diff
@@ -613,8 +613,12 @@ function unwrapPackageName(obj: any, packageName: string): fixturify.DirJSON {
   return getFolder(obj, packageName);
 }
 
+function isError(e: unknown): e is Error {
+  return !!e && typeof e === 'object' && Object.prototype.toString.call(e) === '[object Error]' && 'message' in e;
```

Comment:
it is only null that is typeof 'object', so you can make this `e !== null`

Reply:
Ya makes sense.

Reply:
Discussed offline, but we decided to widen the check to only what we care about, in this case that it's an object with a `code` property.